### ISSUE TITLE
Unexpected behaviour when wrapping ANSI escape codes

### ISSUE THREAD

**Issue by taylorconor** [mgeisler/textwrap#248]
Hello!

I understand that ANSI wrapping support was added in https://github.com/mgeisler/textwrap/issues/140. However adding ANSI codes to the example string from the README produces unexpected line wrapping:
```
fn main() {
    let text = "\u{001b}[31mtextwrap\u{001b}[0m: an efficient and powerful library for wrapping text.";
    println!("{}", textwrap::fill(text, 28));
}
```

produces:
```
textwrap: an
efficient and powerful
library for wrapping text.
```

expected:
```
textwrap: an efficient
and powerful library for
wrapping text.
```

**Comment by mgeisler** [comment]
Hi Conor, nice to see you here :)

That looks weird indeed... I'll have to investigate a bit.

**Comment by mgeisler** [comment]
Thanks for catching this! I'll be fixed in a minute!

**Comment by mgeisler** [comment]
This is fixed in the newly released version 0.13.1.
